lib/rules/camelcase.js

```diff
@@ -266,6 +290,11 @@ module.exports = {
                         report(node);
                     }
 
+                // Check if it's a global variable
+                } else if (ignoreGlobals && isReferenceToGlobalVariable(node)) {
+
+                    // no-op (no-useless-return)
+
```

Comment:
This would still miss some cases, e.g., [this test](https://github.com/eslint/eslint/pull/12782#discussion_r441333485) would fail.

Sorry if it wasn't clear, I was suggesting to add `if (ignoreGlobals && isReferenceToGlobalVariable(node)) { return; }` before the if-else-if chain, like [here](https://github.com/eslint/eslint/blob/7fb45cf13e9908d489bd6d5fba3b7243c01508b9/lib/rules/camelcase.js#L191), similar to `if (isAllowed(name))`.

Reply:
Thank you, now it's clear.